editor/src/clj/editor/asset_browser.clj

```diff
@@ -454,6 +442,16 @@
         (when (and (delete selection) next)
           (select-resource! asset-browser next))))))
 
+(defn replace-template-name
+  ^String [^String template ^String name]
```

Comment:
Not sure we need any of those type hints?

Reply:
Yeah, I added these mostly for documentation purposes. I think it makes sense to do so when the arguments or return values are Java types.